exford/functions.php

```diff
@@ -99,6 +99,9 @@ function exford_setup() {
 				),
 			)
 		);
+
+		// Enable Full Site Editing
+		add_theme_support( 'full-site-editing');
```

Comment:
nit: missing a space
```suggestion
		add_theme_support( 'full-site-editing' );
```